src/Client.cpp

```diff
@@ -183,6 +210,9 @@ void Client::OnAfterCreated(CefRefPtr<CefBrowser> browser)
 
     CEF_REQUIRE_UI_THREAD();
 
+    m_requestHandler->m_messageRouterBrowserSide = CefMessageRouterBrowserSide::Create(constants::messageRouterConfig);
```

Comment:
Actually given the way that the `Manager` processes `Job`s by finding them by `CefBrowser` as long as the `CefMessageRouterBrowserSide` isn't overwritten it should be okay to have only one.